rust-runtime/aws-smithy-runtime/src/test_util/capture_test_logs.rs

```diff
@@ -24,14 +24,25 @@ pub struct LogCaptureGuard(#[allow(dead_code)] DefaultGuard);
 pub fn capture_test_logs() -> (LogCaptureGuard, Rx) {
     // it may be helpful to upstream this at some point
     let (mut writer, rx) = Tee::stdout();
-    if env::var("VERBOSE_TEST_LOGS").is_ok() {
-        eprintln!("Enabled verbose test logging.");
+    let (enabled, level) = match env::var("VERBOSE_TEST_LOGS").ok().as_deref() {
+        Some("debug") => (true, Level::DEBUG),
```

Comment:
it would be nice if you could "loud" this from the API instead of needing to use an environment variable.